### DISCUSSION TITLE
Support for go 1.14

### DISCUSSION THREAD

**Discussion by pohanhuangtw** [shirou/gopsutil#1600] [Q&A]
Hi Community,
Currently I need to use gopsutil to grep the ps result, but the go version must be 1.14 due to some reason.
Any good suggestions?
Thanks.

**Comment by shirou** [comment]
go 1.14 was EoL 3 years ago. I highly recommend updating go version. If you can not, I think around v3.21.2  would work with go 1.14. However, in that case, there may be some problems compared to the latest version.
